examples/planar_vector_tile.js

```diff
@@ -0,0 +1,180 @@
+            type: itowns.STRATEGY_DICHOTOMY,
+        },
+        style: function (properties, feature) {
+            var styles = [];
+            properties.mapboxLayer.forEach(function(layer) {
+                var r = { };
+                // a feature could be used in several layers...
+                if ('paint' in layer) {
+                    if (layer.type == 'fill') {
+                        r.fill = layer['paint']['fill-color'];
+                        r.fillOpacity = layer['paint']['fill-opacity'];
+                    }
+                    if (layer.type == 'line') {
+                        r.stroke = layer['paint']['line-color'];
+                        if ('line-width' in layer['paint']) {
+                            r.strokeWidth = layer['paint']['line-width']['base'];
+                        }
+                        r.strokeOpacity = layer['paint']['line-opacity'];
+                    }
+                }
+                styles.push(r);
+            });
+
+            if (styles.length === 1) {
+                return styles[0];
+            }
+
+            return styles;
+        },
+        filter: function (properties, geometry) {
+            properties.mapboxLayer = [];
+            mapboxLayers.forEach(function(layer) {
+                if (properties.vt_layer !== layer['source-layer']) {
+                    return;
+                }
+                if ('filter' in layer) {
+                    var filteredOut = false;
+                    for (var i = 0; i < layer['filter'].length; i++) {
+                        var filter = layer['filter'][i];
+
+                        if (filter.length === undefined) {
+                            continue;
+                        }
+                        if (filter[0] == '==') {
+                            if (filter[1] == '$type') {
+                                filteredOut |= (filter[2] != geometry.type);
+                            }
+                            else if (filter[1] in properties) {
+                                filteredOut |= (properties[filter[1]] != filter[2]);
+                            }
+                        }
+                        else if (filter[0] == 'in') {
+                            filteredOut |= (filter.slice(2).indexOf(properties[filter[1]]) == -1);
+                        }
+                        if (filteredOut) {
+                            break;
+                        }
+                    }
+                    if (!filteredOut) {
+                        properties.mapboxLayer.push(layer);
+                    }
+                } else {
+                    properties.mapboxLayer.push(layer);
+                }
+            });
+            return properties.mapboxLayer.length > 0;
+        },
+    });
+});
+
+viewerDiv.addEventListener('DOMMouseScroll', onMouseWheel);
+viewerDiv.addEventListener('mousewheel', onMouseWheel);
+
+viewerDiv.addEventListener('mousedown', function mouseDown(event) {
```

Comment:
I suppose you'll want to use `PlanarControls` instead, like you did in #734 ?

Reply:
Yep, after merging 734 I'll remove this here.